client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-content/index.tsx

```diff
@@ -13,20 +15,50 @@ export default function SiteContent(): ReactElement {
 
 	const sites = formatSites( data );
 
-	const columns = {
-		site: translate( 'Site' ),
-		backup: translate( 'Backup' ),
-		scan: translate( 'Scan' ),
-		monitor: translate( 'Monitor' ),
-		plugin: translate( 'Plugin Updates' ),
-	};
+	const columns = [
+		{
+			key: 'site',
+			title: translate( 'Site' ),
+		},
+		{
+			key: 'backup',
+			title: translate( 'Backup' ),
+		},
+		{
+			key: 'scan',
+			title: translate( 'Scan' ),
+		},
+		{
+			key: 'monitor',
+			title: translate( 'Monitor' ),
+		},
+		{
+			key: 'plugin',
+			title: translate( 'Plugin Updates' ),
+		},
+	];
 
 	if ( ! isLoading && ! error && ! sites.length ) {
 		return <div className="site-content__no-sites">{ translate( 'No active sites' ) }</div>;
 	}
+
 	return (
 		<>
 			<SiteTable isFetching={ isLoading } columns={ columns } items={ sites } />
+			<div className="site-content__mobile-view">
+				<>
+					{ isLoading || error ? (
+						<JetpackLogo size={ 72 } className="site-content__logo" />
+					) : (
+						<>
+							{ sites.length > 0 &&
+								sites.map( ( rows: any, index: number ) => (
```

Comment:
You don't need to type these arguments for `.map` callback. TS sets types for those automagically.
```suggestion
								sites.map( ( rows, index ) => (
```